File: 2-semestre/uri-online-judge/c/1072.c
#include <stdio.h>
 
int main() {
    int n, i;
    
    scanf("%d", &n);
    
    int valores[n];
    int in = 0;
    int out = 0;
    
    for (i = 0; i < n; i++) {
        scanf("%d", &valores[i]);
        
        if (valores[i] >= 10 && valores[i] <= 20) in++;
        else out++;
    }
    
    printf("%d in\n", in);
    printf("%d out\n", out);
    
    return 0;
}

File: 2-semestre/uri-online-judge/javascript/1072.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let n = parseInt(lines.shift());
let valores = [];
let valIn = 0;
let valOut = 0;

for (let i = 0; i < n; i++) {
    valores[i] = parseInt(lines.shift());

    if (valores[i] >= 10 && valores[i] <= 20) valIn++;
    else valOut++;
}

console.log(`${valIn} in\n${valOut} out`);
